webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -216,13 +216,11 @@ const HostDetails = ({
             {response && (
               <Text ouiaId="date-text" component={TextVariants.span}>
                 <RelativeDateTime date={response.created_at} defaultValue="N/A">
-                  {date =>
-                    sprintf(__('Created %(date)s by %(owner)s'), {
-                      date,
-                      owner: response.owner_name,
-                    })
-                  }
-                </RelativeDateTime>{' '}
+                  {date => sprintf(__('Created %s'), date)}
+                </RelativeDateTime>
+                {response.creator
+                  ? sprintf(__(' by %s '), response.creator)
```

Comment:
@Ron-Lavi, is my syntax here is ok? (with the spaces at the begging and at the end of the text)

Reply:
I would put the first space in the html and remove the second space since there is no text after the creator